tests/acceptance/features/coreApiSharePublicLink1/changingPublicLinkShare.feature

```diff
@@ -18,253 +18,109 @@ Feature: changing a public link share
     Then the HTTP status code should be "<http-status-code>"
     And as "Alice" file "PARENT/parent.txt" <should-or-not> exist
 
-    @notToImplementOnOCIS @issue-ocis-2079
     Examples:
       | permissions               | http-status-code | should-or-not | public-webdav-api-version |
```

Comment:
Note: this scenario outline will always us the "new" public WebDAV API. So that could be hard-coded at line 17, and remove the public-webdav-api-version column from the examples table.